letsencrypt/config.json

```diff
@@ -20,14 +22,24 @@
     "certfile": "fullchain.pem",
     "keyfile": "privkey.pem",
     "challenge": "http",
-    "dns": {}
+    "dns": {},
```

Comment:
Might be a nitpicking, but you've put these new lines AFTER dns{} here, meaning dns{} sees delta due to the added comma, but you did differently on line 42.  Won't change behavior but it's inconsistent   :-/